hack/build/release-description.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+underline() {
+    echo "$2"
+    printf "%0.s$1" $(seq ${#2})
+}
+
+log() { echo "$@" >&2; }
+title() { underline "=" "$@"; }
+section() { underline "-" "$@"; }
+
+#
+# All sorts of content
+#
+release_notes() {
+    log "Fetching release notes"
+    cat manual-release-notes || echo "FIXME manual notes needed"
```

Comment:
CWD is always the right dir here? No repo-root stuff needed?

Reply:
the script doesn't access other files, so it doesn't need to know cwd